### DISCUSSION TITLE
Upgrade issue

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#8202] [Q&A] [answered]
Hello!

After upgrading Zitadel from 2.52 to 2.54 I've got significantly increased number of DB connections: from 3-5 before the upgrade up to 60-80 after the upgrade. The Postgres CPU usage increased almost 3x times as well.

Can someone point me out to possible reasons for that behavior?

Thank you.

**Comment by ashep** [comment, marked as answer]
The problem is gone after upgrading to v2.55.

**Reply by fforootd** [reply]
Yeah, that was a bug in certain version who did not read the config correctly.

Might relate to #8059
